Configuration/src/Steeltoe.Samples.Configuration/LocalConfigServer.feature

```diff
@@ -1,17 +1,17 @@
-@local_scaffold
-Feature: Simple Configuration
-  In order to show you how to use Steeltoe for simple configurations
-  You can run a some simple configuration samples
-
-  @net6.0
-  Scenario: Simple Configuration (net6.0)
-    When you run: git clone https://github.com/spring-cloud/spring-cloud-config
-    And you run: git -C spring-cloud-config checkout v2.1.4.RELEASE
-    And you run in the background: mvn -f spring-cloud-config/spring-cloud-config-server/pom.xml spring-boot:run
-    And you wait until process listening on port 8888
-    And you set env var ASPNETCORE_ENVIRONMENT to "MyNetCoreEnv"
-    And you run in the background: dotnet run
-    And you wait until process listening on port 5000
-    When you get http://localhost:5000/Home/ConfigServerSettings
-    Then you should see "spring:cloud:config:name = foo"
-    And you should see "spring:cloud:config:env = MyNetCoreEnv"
+@local_scaffold
+Feature: Local Spring Cloud Config Server

```

Comment:
patched up, and tested locally (still have the config server files on disk apparently). It would probably make more sense to either always test against the latest source or use our docker image... Regardless, the pipeline side would need more work to finish this up and I don't think it's the highest priority to have this test right now so I'll just remove the feature file